rclcpp/include/rclcpp/subscription.hpp

```diff
@@ -102,26 +103,37 @@ class Subscription : public SubscriptionBase
     rmw_subscription_t * subscription_handle,
     const std::string & topic_name,
     bool ignore_local_publications,
-    CallbackType callback)
+    CallbackType callback,
+    typename msg_memory_strategy::MessageMemoryStrategy<MessageT>::SharedPtr memory_strategy =
+    msg_memory_strategy::MessageMemoryStrategy<MessageT>::create_default())
   : SubscriptionBase(node_handle, subscription_handle, topic_name, ignore_local_publications),
-    callback_(callback)
+    callback_(callback),
+    message_memory_strategy_(memory_strategy)
   {}
 
+  void set_message_memory_strategy(typename msg_memory_strategy::MessageMemoryStrategy<MessageT>::SharedPtr
```

Comment:
Please wrap after the open parenthesis to have the argument not split but on a single line.